packages/webpack/src/config/base.js

```diff
@@ -251,7 +269,7 @@ export default class WebpackBaseConfig {
     return [
       {
         test: /\.vue$/i,
-        loader: 'vue-loader',
+        loader: require.resolve('vue-loader'),
```

Comment:
Resolving here introduces a potential breaking change for any nuxt module trying to match loader with an exact match.

currently breaking built-in [@nuxt/components](https://github.com/nuxt/components)):
- Usage: https://github.com/nuxt/components/blob/master/src/index.ts#L83 (`find: use => use.loader === 'vue-loader')
- CI: https://github.com/nuxt/nuxt.js/runs/707443791#step:5:36

We may try resolving right before compilation so all hooks are passed.

Reply:
Good catch, so other loaders should be also same? We can add the loaders package resolving right before webpack started by sth like: extendBuild, nuxt hook or webpack hook